test/data/router-methods-helper.js

```diff
@@ -0,0 +1,9 @@
+self.getMethodToTest = () => {
+  const methodRegex = /\?method=(\w*)/;
+  const methodRegexResult = methodRegex.exec(location.search);
```

Comment:
It would be nice to use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams instead, but I guess we need to wait a bit for Chrome 49 to make it to stable.